test/system/screenshots_test.rb

```diff
@@ -69,4 +69,13 @@ class ScreenshotsTest < ApplicationSystemTestCase
     visit node.path
     take_screenshot
   end
+  
+  test 'blog page with location modal' do
```

Comment:
and I hope this works - added an automated screenshot of the modal! Let's see!

Reply:
`Capybara::ElementNotFound: Unable to find css "a.blurred-location-input"`

Reply:
We are not logged in and hence it makes sense that tests are not passing !

https://camo.githubusercontent.com/721dd60bb7945ee8c9212ddef02703969d961e50/68747470733a2f2f73746f726167652e676f6f676c65617069732e636f6d2f706c6f7473322d73637265656e73686f74732f353532322d6661696c757265735f746573745f626c6f675f706167655f776974685f6c6f636174696f6e5f6d6f64616c2e706e67